Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/17_cryoscope_1ns.py

```diff
@@ -163,7 +163,7 @@ def baked_waveform(waveform, pulse_duration):
     # Outer loop for averaging
     with for_(n, 0, n < n_avg, n + 1):
         # Loop over the truncated flux pulse
-        with for_(segment, 0, segment <= const_flux_len + total_zeros, segment + 1):
+        with for_(segment, 1, segment <= const_flux_len + total_zeros, segment + 1):
```

Comment:
If 0 is not part of `square_pulse_segments`, then it should be `with for_(segment, 0, segment < const_flux_len + total_zeros, segment + 1):`